src/request.js

```diff
@@ -1,25 +1,28 @@
 import mitt from 'mitt';
-import {
-  convertBufferToLines
-} from './utils';
+import { convertBufferToLines } from './utils';
 
-const fetcher = Promise
-  .resolve()
-  .then(() => 'fetch' in self ? self.fetch : import('whatwg-fetch').then(() => self.fetch));
+const fetcher = Promise.resolve().then(
+  () =>
+    'fetch' in self ? self.fetch : import('whatwg-fetch').then(() => self.fetch)
+);
 
-export const request = (url, options) => {
+export default (url, options) => {
```

Comment:
I would probably add a name to this function to make stack traces better understood.

Reply:
It will be named by the import into LazyLog.